File: CS/Q212929/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Q212929 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            CreateData();
        }

        private void CreateData() {
            for (int i = 0; i < 10; i++)
                dataTable1.Rows.Add();
        }
    }
}

File: VB/Q212929/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace Q212929
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			CreateData()
		End Sub

		Private Sub CreateData()
			For i As Integer = 0 To 9
				dataTable1.Rows.Add()
			Next i
		End Sub
	End Class
End Namespace